primitives/version/version_test.go

```diff
@@ -1,151 +0,0 @@
-// SPDX-License-Identifier: BUSL-1.1
-//
-// Copyright (C) 2025, Berachain Foundation. All rights reserved.
-// Use of this software is governed by the Business Source License included
-// in the LICENSE file of this repository and at www.mariadb.com/bsl11.
-//
-// ANY USE OF THE LICENSED WORK IN VIOLATION OF THIS LICENSE WILL AUTOMATICALLY
-// TERMINATE YOUR RIGHTS UNDER THIS LICENSE FOR THE CURRENT AND ALL OTHER
-// VERSIONS OF THE LICENSED WORK.
-//
-// THIS LICENSE DOES NOT GRANT YOU ANY RIGHT IN ANY TRADEMARK OR LOGO OF
-// LICENSOR OR ITS AFFILIATES (PROVIDED THAT YOU MAY USE A TRADEMARK OR LOGO OF
-// LICENSOR AS EXPRESSLY REQUIRED BY THIS LICENSE).
-//
-// TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
-// AN “AS IS” BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
-// EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
-// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
-// TITLE.
-
-package version_test
-
-import (
-	"encoding/binary"
-	"testing"
-
-	"github.com/berachain/beacon-kit/primitives/version"
-	"github.com/stretchr/testify/require"
-)
-
-func TestFromUint32(t *testing.T) {
```

Comment:
@calbera looks like these tests were moved to bytes package. Can you confirm?

Reply:
confirmed

Reply:
Also, why we moved this code to bytes? I don't feel strongly about this but I think usually we err towards adding a new type (even if it's just an alias) and adding there its own methods.

Reply:
There's no unique methods on Version once the generics were removed. The Version type alias is still there